e2e/playwright.config.ts

```diff
@@ -0,0 +1,109 @@
+import type { PlaywrightTestConfig } from '@playwright/test'
+import path from 'path'
+
+const e2eReportsFolder = path.resolve(__dirname, 'e2e-reports')
+
+/**
+ * See https://playwright.dev/docs/test-configuration
+ */
+
+const config: PlaywrightTestConfig = {
+  name: 'Podkrepi.bg E2E tests',
+  testDir: './tests',
+  /* Maximum time one test can run for. */
+  timeout: 60 * 1000,
+  expect: {
+    timeout: 10 * 1000,
+  },
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  // TODO update here
+  workers: process.env.CI ? 1 : undefined,
+  // outputDir: path.resolve(e2eReportsFolder, 'output-tests'),
+  reporter: [
+    ['html', { outputFolder: path.resolve(e2eReportsFolder, 'html-report'), open: 'never' }],
+  ],
+  use: {
+    browserName: 'chromium',
+    headless: true,
+    screenshot: {
+      mode: 'only-on-failure',
+      fullPage: true,
+    },
+    video: {
+      mode: 'retain-on-failure',
+      size: {
+        width: 1500,
+        height: 900,
+      },
+    },
```

Comment:
let's collect video only on demand as it will be needed in very rare cases anyway

Reply:
If we delete all artifacts after each execution I don't think they would be a problem. Playwright can record videos using one of the following options: 

- 'off' - Do not record video.
- 'on' - Record video for each test.
- 'retain-on-failure' - Record video for each test, but remove all videos from successful test runs.
- 'on-first-retry' - Record video only when retrying a test for the first time. 

We could just turn the option OFF and use only screenshots.

Reply:
yep, my idea was by default to be "off" and only when investigating a tricky case to switch to "retain-on-failure"

Reply:
https://github.com/podkrepi-bg/frontend/pull/1265/commits/18ec68e1f3646d3f63bc12e297caee77f44f5eef